__tests__/app.test.js

```diff
@@ -31,43 +33,19 @@ describe("GET /api/topics", () => {
 
 describe("GET /api", () => {
   test("Returns with an object describing all the available endpoints", () => {
-    const expectedObject = {
-      "GET /api": {
-        description:
-          "serves up a json representation of all the available endpoints of the api",
-      },
-      "GET /api/topics": {
-        description: "serves an array of all topics",
-        queries: [],
-        exampleResponse: {
-          topics: [{ slug: "football", description: "Footie!" }],
-        },
-      },
-      "GET /api/articles": {
-        description: "serves an array of all articles",
-        queries: ["author", "topic", "sort_by", "order"],
-        exampleResponse: {
-          articles: [
-            {
-              title: "Seafood substitutions are increasing",
-              topic: "cooking",
-              author: "weegembump",
-              body: "Text from the article..",
-              created_at: "2018-05-30T15:59:13.341Z",
-              votes: 0,
-              comment_count: 6,
-            },
-          ],
-        },
-      },
-    };
+    let expectedOutput = null;
+    fs.readFile(`${__dirname}/../endpoints.json`, "utf-8").then(
+      (enpointsDescription) => {
+        expectedOutput = JSON.parse(enpointsDescription);
+      }
+    );
```

Comment:
Because it's JSON, you can just require it in directly then use that variable instead of having to use fs promises. Json files are actually very accessible within js.

This would mean that all you have to do is expect that the required variable is the same as the variable that comes from the server response to simplify even futher!